mkdocs/docs/code-samples/iter8-knative/canary-progressive.md

```diff
@@ -243,7 +243,7 @@ kubectl delete -f $ITER8/samples/knative/quickstart/experimentalservice.yaml
 ```
 
 ??? info "Understanding what happened"
```

Comment:
is this change intentional? I'm still working on the canary-progressive experiment... and the md file needs even more changes (for example describing helm and weight knobs (yet to be) introduced in the experiment. Could we please not make any changes to canary-progressive.md in  this PR?

Reply:
Sure. No problem. I just noticed the numbering was off.